operator/pkg/operator/manifests/banyandb/templates/deployment.yaml

```diff
@@ -0,0 +1,116 @@
+# Licensed to Apache Software Foundation (ASF) under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Apache Software Foundation (ASF) licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Name }}-banyandb
+  namespace: {{ .Namespace }}
+  labels:
+    app: banyandb
+    operator.skywalking.apache.org/banyandb-name: {{ .Name }}
+    operator.skywalking.apache.org/application: banyandb
+    operator.skywalking.apache.org/component: deployment
+spec:
+  replicas: {{ .Spec.Counts }}
+  selector:
+    matchLabels:
+      app: banyandb
+      operator.skywalking.apache.org/banyandb-name: {{ .Name }}
+  template:
+    metadata:
+      labels:
+        app: banyandb
+        operator.skywalking.apache.org/banyandb-name: {{ .Name }}
+        operator.skywalking.apache.org/application: banyandb
+        operator.skywalking.apache.org/component: pod
+    spec:
+      serviceAccountName: {{ .Name }}-banyandb
+
+      {{- if .Spec.Storages }}
+      volumes:
+        {{- range $storage := .Spec.Storages }}
+        - name: {{ $storage.Name }}
+          persistentVolumeClaim:
+            claimName: {{ $storage.Name }}-banyandb
+        {{- end }}
+      {{- end}}
+      containers:
+        - name: banyandb-container
+          image: {{ .Spec.Image }}
+          imagePullPolicy: IfNotPresent
+          args:
+            {{- range $value := .Spec.Config }}
+            - {{ $value }}
+            {{- end }}
+          ports:
+            - containerPort: 17912
```

Comment:
we need to expose 17913 to serve the HTTP endpoint.

Reply:
done